openedx/features/course_experience/templates/course_experience/course-message-fragment.html

```diff
@@ -0,0 +1,32 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import get_language_bidi
+from openedx.core.djangolib.markup import HTML
+from openedx.features.course_experience import CourseHomeMessages
+%>
+
+<%
+is_rtl = get_language_bidi()
+%>
+
+<%block name="content">
```

Comment:
Nit: is this necessary? Isn't the template being called from inside a content block? As this is a fragment, it seems that it should be possible to render it anywhere.